crates/ibc-elc-client-cw/src/context.rs

```diff
@@ -0,0 +1,351 @@
+use crate::error::ContractError;
+use crate::wasm_client_state::WasmClientState;
+use crate::wasm_consensus_state::WasmConsensusState;
+use core::str::FromStr;
+use cosmwasm_std::{Binary, CustomQuery, Deps, DepsMut, Empty, Env, Storage};
+use cw_storage_plus::Map;
+use light_client::types::{Any, ClientId, Height, Time};
+use light_client::Error as LightError;
+use light_client::{ClientReader, HostClientReader, HostContext};
+use prost::Message;
+use store::KVStore;
+
+pub const SUBJECT_PREFIX: &[u8] = b"subject/";
+pub const SUBSTITUTE_PREFIX: &[u8] = b"substitute/";
+
+pub const CLIENT_PREFIX: &str = "clients";
+pub const CLIENT_STATE: &str = "clientState";
+pub const CONSENSUS_STATE_PREFIX: &str = "consensusStates";
+pub const ITERATE_CONSENSUS_STATE_PREFIX: &str = "iterateConsensusStates";
+pub const PROCESSED_TIME: &str = "processedTime";
+pub const PROCESSED_HEIGHT: &str = "processedHeight";
+
+/// - [`Height`] cannot be used directly as keys in the map,
+///   as it doesn't implement some cw_storage specific traits.
+/// - Only a sorted set is needed. So the value type is set to
+///   [`Empty`] following
+///   ([cosmwasm-book](https://book.cosmwasm.com/cross-contract/map-storage.html#maps-as-sets)).
+pub const CONSENSUS_STATE_HEIGHT_MAP: Map<(u64, u64), Empty> =
```

Comment:
Is this map required? Or where is this value referenced?

Reply:
here: https://github.com/datachainlab/ibc-elc-client-cw/pull/1/files#diff-e4bbeeaf31a1434f019a481ac13e09b57ed29d0bbf2afe4fcf302cf2d8a746aaR320

Reply:
removed: https://github.com/datachainlab/ibc-elc-client-cw/pull/1/commits/ea9fd3d893e131c477ecbc7a9c60008549d40be1